addons/mail/models/mail_activity_plan_template.py

```diff
@@ -25,14 +25,13 @@ class MailActivityPlanTemplate(models.Model):
     )
     summary = fields.Char('Summary', compute="_compute_summary", store=True, readonly=False)
     responsible_type = fields.Selection([
-        ('on_demand', 'On demand'),
-        ('other', 'Other'),
-    ], default='on_demand', string='Responsible', required=True)
+        ('on_demand', 'Ask at launch'),
+        ('other', 'Default user'),
+    ], default='on_demand', string='Assignation', required=True)
```

Comment:
Isn't it "assignment" the right word ? Assignation is used notably for lovers :D

Reply:
Yes, you’re tight, thanks